### PR TITLE
Update mongodb reference to fix unresolved images

### PR DESCRIPTION
**Pull request by gmontalvoy**
References to MongoDB 2.6 is no longer working as there is no such tag references on openshift project at 4.4

### CODE REVIEW

**Comment by wkulhanek** [comment]
I think we want a new file. Otherwise merging this will break the existing class (which runs on 4.1).

**Comment by gmontalvoy** [comment]
I also updated wildfly files

File: mlbparks-template-eap copy-4.4.json [hunk 1]
```diff
@@ -0,0 +1,463 @@
+{
```
Comment (newgoliath) [inline mlbparks-template-eap copy-4.4.json]: sure the file should be named `*.copy-4.4.json` ?  `s/copy//g` ?

**Review by newgoliath — APPROVED** [review]

File: mlbparks-template-eap copy-4.4.json (hunk 1 above)
Comment (gmontalvoy) [reply mlbparks-template-eap copy-4.4.json]: Fixed, so much thank you for review!!

**Review by newgoliath — APPROVED** [review]
LGTM

**Pull request merged**